kiteconnect/connect.py

```diff
@@ -794,8 +797,8 @@ def _parse_instruments(self, data):
             row["lot_size"] = int(row["lot_size"])
 
             # Parse date
-            if len(row["expiry"]) == 10:
```

Comment:
Again can't we pass this to `_format_response` and handle this?

Reply:
Should we continue using earlier string length comparison(len(resp["login_time"]) == 19) inside _format_response ?

Reply:
similar to above, just add the field to - https://github.com/zerodha/pykiteconnect/blob/bacaafc83b38655f94c86c4abb03682ee9facc7e/kiteconnect/connect.py#L402

Reply:
Done.